string name
string acceptor
---
bool added
